pkg/csi/utils/utils.go

```diff
@@ -0,0 +1,39 @@
+// This file is part of MinIO DirectPV
+// Copyright (c) 2021, 2022, 2023 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package controller
+
+import (
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	directpvtypes "github.com/minio/directpv/pkg/apis/directpv.min.io/types"
+	"github.com/minio/directpv/pkg/k8s"
+	"github.com/minio/directpv/pkg/types"
+)
+
+// GetCSIVolumeCondition returns the CSI volume condition
+func GetCSIVolumeCondition(volume *types.Volume) *csi.VolumeCondition {
```

Comment:
Looks like this function is not used anywhere. If you use this in this PR, make this function as method to `types.Volume` and remove this file